### DISCUSSION TITLE
Racking Server

### DISCUSSION THREAD

**Discussion by oosterma** [netbox-community/netbox#14294] [Help Wanted!]
Hi,

I have a question concerning racking servers.
At the moment I have a server that has status decommisioned and still in the rack.
I would like to place another server in the same spot but with status planned. Netbox doesn't seem to be allowing this...
How do you guys do this. Because it seems to me as a normal server lifecycle.

**Comment by candlerb** [comment]
You can't do this with Netbox today. One or other server has to be "removed" from the rack.

For large data centre migrations, I plan the new locations in a spreadsheet, and after the migration update the positions in Netbox.

**Comment by gdprdatasubect** [comment]
You could add a reservation and use a custom field for those to indicate which one should go in there.

The "Object: A single NetBox object of the type defined by object_type" seems pretty fitting if you want to go that route
